apps/website/locales/ar.ts

```diff
@@ -0,0 +1,283 @@
+import { defineI18nLocale } from false;
+export default defineI18nLocale(() => ({
```

Comment:
Wouldn't it be better to do this? (1/2)
```suggestion
const defineI18nLocale = (() => ({
```